VDA-StorageNode-Registry/ChangeLog.md

```diff
@@ -1,3 +1,12 @@
+2024-01-18 (V1.3.0)
```

Comment:
This code hasn't been officially released, so shouldn't have a version number yet.

You can just track all the changes without any version numbers or dates until we do an official release.

Reply:
Just then only the dates in the `ChangeLog.md`?

Reply:
Set package version as 1.0.0 in the `package.json` and `ChangeLog.md`.